### PR TITLE
[@types/react-gauge-chart] update types

### PR DESCRIPTION
**Pull request by meirkl**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/Martin36/react-gauge-chart/pull/61
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/react-gauge-chart/react-gauge-chart-tests.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
 import * as React from 'react';
 import GaugeChart from 'react-gauge-chart';
 
-const Gauge: React.ReactElement = <GaugeChart id="0" />;
+const Gauge: React.ReactElement = <GaugeChart />;
```
Comment (peterblazejewicz) [inline types/react-gauge-chart/react-gauge-chart-tests.tsx:4]:
```suggestion
let Gauge: React.ReactElement = <GaugeChart id="gauge-chart1" />;
Gauge = <GaugeChart />;
```
let's have both, for readme-driven-development tests, OK?

Also in our PR to correct props types in package, there is no change for `id` described as `required`,
what's the status of `id`?
For the readme-driven-development, it's required and you should modify your props.
thx!

Comment (meirkl) [reply types/react-gauge-chart/react-gauge-chart-tests.tsx:4]: According to [PR](https://github.com/Martin36/react-gauge-chart/pull/37) the id is optional.

Will open a PR to update the docs.
Thanks!

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@meirkl thx!

**Comment by meirkl** [comment]
Ready to merge

**Pull request merged**